sdk/storage/Azure.Storage.Files.DataLake/tests/DirectoryClientTests.cs

```diff
@@ -1557,6 +1557,40 @@ public async Task RenameAsync_DifferentSasUri()
             Response<PathProperties> response = await destFile.GetPropertiesAsync();
         }
 
+        [LiveOnly]
```

Comment:
Why live only?

Reply:
So our other SAS tests are also marked as LiveOnly. (I'm not sure if I was the one who did that)

I assumed that has something to do with the sv=version messing up the recordings every time we go to update the SAS. (Unless this issue is already fixed?)